### ISSUE TITLE
Hide and Show Toolbar with state

### ISSUE THREAD

**Issue by nelreina** [mkhstar/suneditor-react#12]
Thank you for this awesome library. I was looking around several options for editors , I find this one to fit my use case. But it is not clear to me how I can  dynamically  hide and show the toolbar / or the editor box. 
 
I want to do like multiple section and put an edit button aside that my users can click to edit the section.
On disable state it should not show the toolbar and the box just the content. 

Is this posible ?

**Comment by mkhstar** [comment]
Hi you could use the `show` or `disable` props as specified in the documentation.

For example the code below toggles between enable and disable when the button is clicked

```jsx

import React, {useState} from "react";
import SunEditor from 'suneditor-react';
import 'suneditor/dist/css/suneditor.min.css';

export default function App() {
  const [state, setState] = useState(false);
  return (
    <div className="App">
      <button onClick={() => setState(!state)}>Click Me</button>
     <SunEditor disable={state}/>
    </div>
  );
}


```

**Comment by nelreina** [comment]
Yes I tried that but It does not hide the Toolbar

Look at this Jodit example [Toolbar](https://xdsoft.net/jodit/play.html?uploader%5BinsertImageAsBase64URI%5D=true&showCharsCounter=false&showWordsCounter=false&showXPathInStatusbar=false)
This is what I want to achieve...

 
![Screen Shot 2020-01-11 at 4 30 13 PM](https://user-images.githubusercontent.com/5773049/72210332-11fdc480-3490-11ea-8708-2bd753459a2a.png)

![Screen Shot 2020-01-11 at 4 31 33 PM](https://user-images.githubusercontent.com/5773049/72210335-17f3a580-3490-11ea-8460-b8caa0f103ba.png)

**Comment by mkhstar** [comment]
I think this should be a feature request for @JiHong88 suneditor.

**Comment by JiHong88** [comment]
Hi, @nelreina 

It can be implemented using the object returned by the construction function.
```javascript
...
const editorObject = ...

// hide
editorObject.core.context.element.toolbar.style.display = 'none'
editorObject.core.context.element._stickyDummy.style.display = 'none'
// show
editorObject.core.context.element.toolbar.style.display = ''
editorObject.core.context.element._stickyDummy.style.display = ''
```

@mkhstar returning an object from suneditor-react?

**Comment by JiHong88** [comment]
Hi, @nelreina @mkhstar 

Toolbar-related methods have been added. [#functions](https://github.com/JiHong88/SunEditor#functions)
 - editor.toolbar.disabled()
 - editor.toolbar.enabled()
 - editor.toolbar.hide()
 - editor.toolbar.show()

Updated in [v2.24.0](https://github.com/JiHong88/SunEditor/releases/tag/2.24.0)

@mkhstar  can you update suneditor-react to the latest version?

**Comment by mkhstar** [comment]
Alright Thanks @JiHong88

**Comment by mkhstar** [comment]
implemented in version [v2.5.0](https://www.npmjs.com/package/suneditor-react)
